index.html

```diff
@@ -76,6 +76,17 @@ <h2>CapturedMouseEvent interface</h2>
         more scrutiny in which information may safely be exposed. At present, only information
         already available to the capturing application is exposed - the mouse coordinates.
       </p>
+      <p>
+        For any <a data-cite="screen-capture/#dfn-capture-session">capture-session</a>
+        that has an associated {{CaptureController}}, events with type
```

Comment:
OK. Do you mean to remove reference to a capture session too? (I'm not sure if "capture session" means that the source is not ended, but if so that would be redundant).

Reply:
> Do you mean to remove reference to a capture session too?

I have no problem with referencing it, but I think my proposal clarifies when events start being dispatched and when we stop dispatching them.